cdap-unit-test/src/test/java/co/cask/cdap/spark/metrics/SparkMetricsIntegrationTestRun.java

```diff
@@ -44,7 +45,7 @@
 @Category(XSlowTests.class)
 public class SparkMetricsIntegrationTestRun extends TestFrameworkTestBase {
 
-  public static final String METRICS_KEY = "system.driver.BlockManager.memory.remainingMem_MB";
+  private static final String METRICS_KEY = "system.<driver>.BlockManager.memory.remainingMem_MB";
```

Comment:
Spark 1.2 emits as "driver.*", while Spark 1.3 emits as "<driver>.*". Not sure if we should normalize it or not. Thoughts?